Rules/CTF/Scripts/CTF.as

```diff
@@ -577,6 +582,65 @@ shared class CTFCore : RulesCore
 		}
 	}
 
+	void CheckStalemate()
```

Comment:
afaict this could all be just on server + sync required stuff to players? seems to do that anyway

Reply:
It would complicate it horribly to just do it server-side and send it, plus the check has minimal performance impact and saves bandwidth~